components/Layout.js

```diff
@@ -0,0 +1,90 @@
+import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/legacy/image";
+
+import "../public/logo.png";
+
+const Layout = ({children}) => {
+  return (
+    <>
+      <Navbar>
+        <Logo src="/logo.png" alt="kidsFi logo" width={80} height={80} />
+        <StyledList>
+          <Link href="">
+            <svg
```

Comment:
Please consider a `<title>` tag for your SVGs for accessibility (https://css-tricks.com/accessible-svgs/)